cli/popper/gha.py

```diff
@@ -491,7 +491,7 @@ def docker_create(self, img):
         env_vars.update({'HOME': os.environ['HOME']})
         volumes = [self.workspace, os.environ['HOME'], '/var/run/docker.sock']
         if self.debug:
-            pu.info('DEBUG: Invoking docker_create() method\n')
+            pu.debug_log('Invoking docker_create() method\n')
```

Comment:
now that we are using `logging`, we can now remove the conditionals and do `log.debug` directly (assuming we will remove the wrappers in `utils.py` as mentioned below)